dev_tools/configure_vscode_for_bazel.py

```diff
@@ -35,6 +35,11 @@ def parse_arguments(argv: Optional[Sequence[str]] = None) -> argparse.Namespace:
         action="store_true",
         help="Don't ask for confirmation.",
     )
+    parser.add_argument(
+        "--bazel-workspace",
+        type=Path,
+        help="Path to the Bazel workspace. By default, it's determined by running `bazel info workspace`.",
+    )
```

Comment:
This should not be required. For testing, you can always do a `pip3 install -e .` from the repo root

Reply:
Done, removed that part from the PR.